examples/llm/mcloud/mcli-hf-generate.yaml

```diff
@@ -0,0 +1,24 @@
+integrations:
+- integration_type: git_repo
+  git_repo: mosaicml/examples
+  git_branch: release/v0.0.4 # use your branch
+  # git_commit: # OR use your commit hash
+  pip_install: -e .[llm]
+  ssh_clone: false # Should be true if using a private repo
+
+command: |
+  cd examples/examples/llm/inference
+  pip install awscli
```

Comment:
change to generic download as discussed

Reply:
Sadly not possible to download a folder right now generically since we can't `ls` the object store.